### ISSUE TITLE
Implement Testing

### ISSUE THREAD

**Issue by petehamilton** [petehamilton/citier#3]
Need to implement some testing so changes can be checked to ensure they don't break code functionality.

**Comment by dmitry** [comment]
Absolutely. Cannot use this, because I can't be sure that it will work in the future rails versions.

**Comment by hidde-jan** [comment]
We are tying to see if we can use citier in an app that also uses devise. I did a fork to make sure they play along nicely (they didn't at first), but I have no idea if I may have broken something.

We will probably write some tests and, if we do, contribute them back.
